core/src/main/scala/com/avast/dapper/DefaultCassandraDao.scala

```diff
@@ -0,0 +1,76 @@
+package com.avast.dapper
+
+import java.util.concurrent.Executor
+
+import com.datastax.driver.core._
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DefaultCassandraDao[PrimaryKey, Entity <: CassandraEntity[PrimaryKey]](session: Session)(
+    implicit entityMapper: EntityMapper[PrimaryKey, Entity],
+    ec: ExecutionContext,
+    ex: Executor)
+    extends CassandraDao[PrimaryKey, Entity]
+    with LazyLogging {
+
+  import DefaultCassandraDao._
+
+  override def tableName: String = entityMapper.tableName
+
+  override def get(primaryKey: PrimaryKey, queryOptions: ReadOptions = ReadOptions.Default): Future[Option[Entity]] = {
+    logger.debug(s"Querying $tableName with primary key $primaryKey and $queryOptions")
+
+    val convertedKey = entityMapper.convertPrimaryKey(primaryKey)
+    val st = new SimpleStatement(s"select * from $tableName where ${entityMapper.primaryKeyPattern}", convertedKey: _*)
```

Comment:
I would feel much safer if these statements were created by `QueryBuilder` instead of string concatenation.

Reply:
Not that easy... I'll try.